Fix the DES library alignment, which was otherwise causing problems on
SPARC Linux.  Pulled from the upstream stable branch, will be in 1.4.5.

--- openafs.orig/src/des/des.h
+++ openafs/src/des/des.h
@@ -20,7 +20,10 @@
 typedef unsigned char des_cblock[8];	/* crypto-block size */
 /* Key schedule */
 typedef struct des_ks_struct {
-    des_cblock _;
+    union {
+        afs_uint32 i[2];
+        des_cblock _;
+    } _;
 } des_key_schedule[16];
 
 #define DES_KEY_SZ 	(sizeof(des_cblock))
--- openafs.orig/src/des/pcbc_encrypt.c
+++ openafs/src/des/pcbc_encrypt.c
@@ -24,9 +24,9 @@
 #ifndef KERNEL
 #include <stdio.h>
 #endif
-#include <des.h>
 #include <afsconfig.h>
 #include <afs/param.h>
+#include <des.h>
 #include "des_prototypes.h"
 
 RCSID
